docs/reference/index.rst

```diff
@@ -11,3 +11,4 @@ Reference
     hooks
     signals
     project_template
+    multi_site_capabilities
```

Comment:
This isn't really reference documentation as such - it would probably fit better under 'advanced topics'. (Admittedly, there are quite a few existing places in the docs where the line between informational and reference docs is blurrier than it should be :-) )